src/rdkafka_conf.c

```diff
@@ -306,6 +306,7 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
                         },
                         { 0x400, "zstd", _UNSUPPORTED_ZSTD },
                         { 0x800, "sasl_oauthbearer", _UNSUPPORTED_SSL },
+                        { 0x1000, "ssl_client_cert_callback", _UNSUPPORTED_OPENSSL_1_0_2 },
```

Comment:
Don't think we need to add this as a feature flag, has little value for most users and the fetch_cb setter should just return NOT_SUPPORTED instead.